src/prpy/util.py

```diff
@@ -446,7 +482,8 @@ def quadraticObjective(dq, *args):
 
 
 def ComputeJointVelocityFromTwist(robot, twist,
-                                  objective=quadraticObjective,
+                                  objective=quadraticPlusJointLimitObjective,
```

Comment:
I think `quadraticObjective` should be default in `prpy.util`. There are way too many parameters in `quadraticPlusJointLimitObjective` for this to be easily generalizable to all robots. If you feel like it, you should use the other objective with specially tuned parameters within a specific `robot.py`.